README.md

```diff
@@ -2,91 +2,87 @@
 
 # Getting started
 
-Welcome to the getting started with [Utrecht University GitHub organization page](https://github.com/UtrechtUniversity). A GitHub organization is a shared account where members (UU employees) can manage their projects using their personal GitHub account. The UtrechtUniversity Github organization is meant for storing, managing and publishing research projects. Below you will find instructions for connecting to the organization and general usage instructions for the organization. See the bottom of the page for resources to get started with using Git for version control.
+Welcome to the getting started with [Utrecht University GitHub organization page](https://github.com/UtrechtUniversity). A GitHub organization is a shared account where members can manage their projects using their personal GitHub account. The Utrecht University GitHub organization is meant for storing, managing and publishing research projects. Below you will find instructions for connecting to the organization and general usage instructions for the organization. See the bottom of the page for resources to get started with using Git for version control.
 
 For best practices, see [Best Practices for Git @UtrechtUniversity](https://github.com/UtrechtUniversity/best-practices).
 
-## First time connecting to UtrechtUniversity GitHub
+## First time connecting to the GitHub organization
 
-You can connect to the Utrecht University GitHub organization when you are an UU employee only. Connect using your personal existing GitHub account. During this proces you will have to authenticate via your SolisID. The aim of this is to get access to the organization, however after access you will still be working under your personal GitHub account. By using your personal GitHub account all your contributions in different organizations and personal repositories can be displayed on your personal profile page which is your Github CV.
+Membership of the Utrecht University GitHub organization, including access to internal repositories, is restricted to employees of Utrecht University (UU). You can use your existing personal GitHub account; if you do not have an account yet, please create one. During this process, you will have to authenticate your personal GitHub account by using your Solis-id. Your Solis-id is merely used to verify your employee status; all commits and actions performed in GitHub will be attributed to your personal GitHub account. By using one GitHub account, your GitHub profile page (which can be used as your curriculum vitae) will not only show your contributions in repositories belonging to this GitHub organization, but contributions in different GitHub organizations or personal repositories as well.
```

Comment:
Suggestion: Would it be better to change the authentication sentence to:
"you will have to authenticate for this organization using your Solis-id"?
I think it is important for the reader to know that the solis id is only needed for the organization and will not alter any other normal interactions with e.g. personal repositories in github

Reply:
Fair point, I'll change that.